docs/09-Advanced.md

```diff
@@ -125,6 +125,19 @@ myLineChart.getDatasetAtEvent(e);
 // => returns an array of elements
 ```
 
+#### .getDatasetMeta(index)
+
+Looks for the dataset that matches the current index and returns that metadata. This returned data has all of the metadata that is used to construct the chart, and can be used to share data between charts.
```

Comment:
I may not understand this sentence correctly, but it should *not* be used to share data between charts because `getDatasetMeta(index)` returns meta data for the current chart. This has been implemented to allow charts to share the same user data.